docs/reference/glossary-of-terms.md

```diff
@@ -0,0 +1,227 @@
+
+Acknowledgement: Many terms are taken directly from [OAK documentation](https://incatools.github.io/ontology-access-kit/glossary.html) with the permission of Chris Mungall. Many descriptions are also taken from https://www.w3.org/TR/owl2-syntax/
+
+#### Annotation
+This term is frequently ambiguous. It can refer to [Text Annotation](#Text-Annotation), [OWL Annotation](#OWL Annotation), or [Association](#Association).
+
+#### AnnotationProperty
+Annotation properties are used to place annotations on individuals, class names, property names, and ontology names. They do not affect the logical definition unless they are used as a "shortcut" that a pipeline expands to a logical axiom.
+
+#### Anonymous Ancestor
+An accumulation of all of the superclasses from ancestors of a class.
+
+#### Anonymous Individual
+If an individual is not expected to be used outside an ontology, one can use an anonymous individual, which is identified by a local node ID rather than a global IRI. Anonymous individuals are analogous to blank nodes in RDF.
+
+#### API
+Application Programming Interface. An intermediary that allows two or more computer programs to communicate with each other. In ontologies, this usually means an [Endpoint](#Endpoint) in which the ontology can be programmatically accessed.
+
+#### Application Ontology
+Usually refers to a [Project Ontology](#Project Ontology).
+
+#### Axiom
+Axioms are statements that are asserted to be true in the domain being described. For example, using a subclass axiom, one can state that the class a:Student is a subclass of the class a:Person.
+
+#### Bioportal
+An [Ontology Repository](#Ontology-Repository) that is a comprehensive collection of multiple biologically relevant ontologies.
+
+#### Concept
+See [Term](#Term)
+
+#### Controlled Vocabulary
+Standardized and organized arrangements of words and phrases that provide a consistent way to describe data. A controlled vocabulary may or may not include definitions. Ontologies can be seen as a controlled vocabulary expressed in an ontological language which includes relations.
+
+#### Class
+An [Ontology elements](#Ontology-element) that formally represents something that can be instantiated. For example, the class "heart"
+
+#### CURIE
+A CURIE is a compact [URI](#URI). For example, `CL:0000001`.
+
+#### Data Model
+An abstract model that organizes elements of data and standardizes how they relate to one another and to the properties of real-world entities.
+
+#### DatatypeProperty
+DatatypeProperty relate terms to literal data (e.g., strings, numbers, datetimes, etc.) as opposed to [ObjectProperty](#ObjectProperty) which relate individuals to other terms. 
```

Comment:
Doesn't sufficiently differentiate from AnnotationProperty which can also relate terms (better to say Ontology element or OWL Entity?) to strings, number, datatimes, etc).  The difference is that Data  Property axoims are on the logical side of OWL - usable by reasoners.  Annotation property axioms are not.